roles/dns/tasks/dns_common.yml

```diff
@@ -24,6 +24,18 @@
     regexp: "allow-query     { localhost; };"
     replace: "allow-query     { any; };"
     backup: yes
+- name: Configure bind to not use dnssec
+  replace:
+    path: /etc/named.conf
+    regexp: "dnssec-enable yes;"
+    replace: "dnssec-enable no;"
+    backup: yes
+- name: Configure bind to not use dnssec
```

Comment:
Can we have a different name for this task; it's currently identical to the one above.